config/manager/manager.yaml

```diff
@@ -54,6 +54,9 @@ spec:
             value: quay.io/konveyor/kubevirt-velero-plugin:v0.2.0
           - name: RELATED_IMAGE_MUSTGATHER
             value: registry.redhat.io/oadp/oadp-mustgather-rhel8:v1.2
+          # TODO change
+          - name: RELATED_IMAGE_NON_ADMIN_CONTROLLER
```

Comment:
Did we setup auto builds in the quay konveyor/migtools org for the NAC repo ? If not then thats the first thing we need to do for this PR to work I believe.

Reply:
no, that is indeed a blocker

Before setting up PROW job, just want to confirm image name. Once settled, I will create it